### ISSUE TITLE
RuntimeError: /home/ader/.config/google-chrome/Default/Favicons is version 36

### ISSUE THREAD

**Issue by D3m0nKingx** [bleachbit/bleachbit#226]
```bash
Error in execution of <bleachbit.Command.Function instance at 0x7f7b3c60dfc8>
Traceback (most recent call last):
  File "/usr/share/bleachbit/bleachbit/Worker.py", line 84, in execute
    for ret in cmd.execute(self.really_delete):
  File "/usr/share/bleachbit/bleachbit/Command.py", line 146, in execute
    self.func(self.path)
  File "/usr/share/bleachbit/bleachbit/Special.py", line 161, in delete_chrome_favicons
    raise RuntimeError('%s is version %d' % (path, ver))
RuntimeError: /home/ader/.config/google-chrome/Default/Favicons is version 36
debug: DeepScan.scan: searches= {'/home/ader': [u'^\\.DS_Store$', u'\\.[Bb][Aa][Kk]$', u'[a-zA-Z]{1,4}~$', u'^~wr[a-z][0-9]{4}\\.tmp$', u'^ppt[0-9]{4}\\.tmp$', u'^Thumbs\\.db$', u'^Thumbs\\.db:encryptable$']}
```

**Comment by az0** [comment]
Which version of BleachBit are you using?  You can find it by clicking **Help - About**.

**Comment by D3m0nKingx** [comment]
BleachBit version 1.10
Copyright (C) 2008-2015 Andrew Ziem.  All rights reserved.

Installed using `sudo apt install bleachbit` on Ubuntu 16.04.3 LTS

**Comment by az0** [comment]
Ubuntu does not keep up to date with the releases of BleachBit. (Also, they update the notification of new releases.) This should have been fixed May 2017 for BleachBit 1.12.

Please update to [BleachBit 1.12 stable](https://www.bleachbit.org/download), [1.17 beta](https://www.bleachbit.org/news/bleachbit-117-beta), or later. 

If you still have the same issue with the newer release, please reopen this ticket.
